### ISSUE TITLE
`libjpeg` and `libpng` are not installed with `@now/node`

### ISSUE THREAD

**Issue by sto3psl** [zeit/now-cli#1925]
Hi everyone and thank you for this amazing project! I use `now` since some time but am now running into a small issue with `now/node`.

I'm building a service that optimizes and converts images on the fly and am using [imagemin](https://github.com/imagemin/imagemin) and [imagemin-webp](https://github.com/imagemin/imagemin-webp). Deployment to now.sh works just fine as seen here (https://imagemin-webp-now-d5mnaefey.now.sh/img/sto3psl.png).

The code I'm deploying can be found in this repository -> https://github.com/sto3psl/imagemin-webp-now

Locally everything works but after deploying to now.sh, I get the following errors for `jpg` and `png` source files respectively:

```
curl 'https://imagemin-webp-now-d5mnaefey.now.sh/?img=https://imagemin-webp-now-d5mnaefey.now.sh/img/sto3psl.jpeg'

JPEG support not compiled. Please install the libjpeg development package before building.
Error! Could not process file /tmp/355075bf-32c8-41b0-b701-34b2e8bb8b01
Error! Cannot read input picture file '/tmp/355075bf-32c8-41b0-b701-34b2e8bb8b01'
```

```
curl 'https://imagemin-webp-now-d5mnaefey.now.sh/?img=https://imagemin-webp-now-d5mnaefey.now.sh/img/sto3psl.png'

PNG support not compiled. Please install the libpng development package before building.
Error! Could not process file /tmp/5b9e21e7-e9f8-494d-8668-feaf45a69a7f
Error! Cannot read input picture file '/tmp/5b9e21e7-e9f8-494d-8668-feaf45a69a7f'
```

`webp` input files work just fine and serve the `webp image`.

```
curl 'https://imagemin-webp-now-d5mnaefey.now.sh/?img=https://imagemin-webp-now-d5mnaefey.now.sh/img/sto3psl.webp'
```

It seems like the lambda doesn't have `libpng` and `libjpeg` installed. Is there anything I can do to fix that. I have seen similar issues with a dependency on `sharp` but couldn't figure out what to do there. It would be super nice to have `imagemin-webp` supported with `@now/node`.

Let me know if you need more information and thanks you in advance!

**Comment by kainy** [comment]
I meet the same problem。。

**Comment by yuriploc** [comment]
I see the same error when deploying a gatsby website.

**Comment by juliangruber** [comment]
@yuriploc there's a gastby example in https://github.com/zeit/now-examples/tree/master/gatsby that works with Now, is there maybe a plugin that requires those libs to be present?

**Comment by yuriploc** [comment]
> @yuriploc there's a gastby example in https://github.com/zeit/now-examples/tree/master/gatsby that works with Now, is there maybe a plugin that requires those libs to be present?

@juliangruber Yes. That lib must be installed locally, it's not bundled with the npm lib anymore. I can build locally, but now lambdas can't build it. More info here https://github.com/zeit/now-builders/issues/329

**Comment by jclif** [comment]
Having trouble with this as well, when using `next-optimized-images`.
